style.css

```diff
@@ -458,3 +473,10 @@ input {
     font-size: 1rem;
   }
 }
+
+@media screen and (min-width: 700px) and (max-width: 960px) {
+  .portfoilio-wrapper {
+    grid-template-columns: repeat(2, 1fr);
+    gap: 1rem;
+  }
+}
```

Comment:
- [ ] Kindly make sure you use the the same `typo`, `width`, `color` , `font` as the design provided in figma.
See your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/192849384-d71791a6-3017-4a06-8396-5ad5bf6e524f.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/192849519-7337ad24-ea9f-40bd-9bc3-34cc022b5208.png)

Reply:
Kindly, I suggest you follow the instructions of the previous reviewer, change the lines and icons from blue to gray